### PR TITLE
Eval: run a code string

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by odsantos** [comment]
Hi @ashuradev would you please resolve the conflicts or inform if you're unavailable to continue with this PR? Thank you.

**Comment by ashuradev** [comment]
> Olá @ashuradev, poderia resolver os conflitos ou informar se não estiver disponível para continuar com este PR? Obrigado.

I resolved conflicts :)

File: 1-js/99-js-misc/02-eval/article.md [hunk 1]
```diff
@@ -46,69 +46,69 @@ function f() {
...
+**Se o código avaliado precisa de variáveis locais, mude `eval` para `new Function` e passe-os como argumentos:**
 
 ```js run
 let f = new Function('a', 'alert(a)');
 
 f(5); // 5
 ```
 
-The `new Function` construct is explained in the chapter <info:new-function>. It creates a function from a string, also in the global scope. So it can't see local variables. But it's so much clearer to pass them explicitly as arguments, like in the example above.
+A construção `new Function` é explicada no capítulo <info:new-function>. Ele cria uma função a partir de uma string, também no escopo global. Portanto, ele não pode ver variáveis locais. Mas é muito mais claro passá-los explicitamente como argumentos, como no exemplo acima.
```
Comment (odsantos) [inline 1-js/99-js-misc/02-eval/article.md:106]:
```suggestion
A construção `new Function` é explicada no capítulo <info:new-function>. Ela cria uma função a partir de uma string, também no escopo global. Portanto, ele não pode ver variáveis locais. Mas é muito mais claro passá-las explicitamente como argumentos, como no exemplo acima.
```

File: 1-js/99-js-misc/02-eval/article.md [hunk 2]
```diff
@@ -46,69 +46,69 @@ function f() {
...
 
-## Summary
+## Resumo
 
-A call to `eval(code)` runs the string of code and returns the result of the last statement.
-- Rarely used in modern JavaScript, as there's usually no need.
-- Can access outer local variables. That's considered bad practice.
-- Instead, to `eval` the code in the global scope, use `window.eval(code)`.
-- Or, if your code needs some data from the outer scope, use `new Function` and pass it as arguments.
+Uma chamada para `eval(codigo)` executa a string de código e retorna o resultado da última instrução.
```
Comment (odsantos) [inline 1-js/99-js-misc/02-eval/article.md:110]:
```suggestion
Uma chamada para `eval(code)` executa a string de código e retorna o resultado da última instrução.
```

File: 1-js/99-js-misc/02-eval/article.md [hunk 3]
```diff
@@ -46,69 +46,69 @@ function f() {
...
+- Raramente usado no JavaScript moderno, pois geralmente não há necessidade.
+- Pode acessar variáveis locais externas. Isso é considerado uma má prática.
+- Em vez disso, para `eval` o código no escopo global, use `window.eval(codigo)`.
```
Comment (odsantos) [inline 1-js/99-js-misc/02-eval/article.md:113]:
```suggestion
- Em vez disso, para `eval` o código no escopo global, use `window.eval(code)`.
```

**Review by odsantos — CHANGES_REQUESTED** [review]
Hi @ashuradev here is a review.
Would you please let me know of something you might disagree?
Thank you,
Osvaldo.

**Review by odsantos — APPROVED** [review]
Hi @ashuradev , 

I hope you don't mind I added three minor changes I believe I forgot to ask.
I suppose we still need a second reviewer.

Thank you very much.

**Comment by ashuradev** [comment]
Okay :)

**Review by eujuliu — APPROVED** [review]

**Comment by odsantos** [comment]
@jjuliomarttins , thank you.

**Comment by odsantos** [comment]
Hello @luizclr , @adriavieira314 , @kiniggit , @Mr-Saxobeat , please this  PR #159 needs a second review.  Thank you.

**Comment by odsantos** [comment]
Hello @luizclr , @jonnathan-ls  ,  @Mr-Saxobeat ,  this PR https://github.com/javascript-tutorial/pt.javascript.info/pull/159 needs a second review. Thank you.

**Comment by odsantos** [comment]
The account was deleted.
Much appreciated for all done.
I'm taking this work and creating a new PR.
Best wishes.

**Pull request closed without merging**